sklearn/svm/classes.py

```diff
@@ -518,6 +518,14 @@ class SVC(BaseSVC):
         .. versionchanged:: 0.17
            Deprecated *decision_function_shape='ovo' and None*.
 
+    break_ties : bool, optional (default=False)
+        If true, `decision_function_shape`='ovr', and number of classes > 2,
+        `predict` will break ties according to the confidence values of
+        `decision_function`. Please note that breaking ties comes
+        at a relatively high computational cost compared to a simple predict.
```

Comment:
the practical utility from this explanation may not be clear to everybody

Reply:
also, documentation of the default=False case is missing